frontend/scripts/index.js

```diff
@@ -45,8 +47,24 @@ function addAttendance(attendance_ID) {
     xmlHttp.setRequestHeader("Content-Type", "application/json");
     xmlHttp.send(JSON.stringify(attendance_json));
     alert(xmlHttp.responseText);
+    return xmlHttp;
```

Comment:
Why are we returning the http object? If we need to return anything, ideally we should just return the body (JSON dictionary)